test/e2e/installplan_e2e_test.go

```diff
@@ -3122,6 +3217,18 @@ var _ = Describe("Install Plan", func() {
 		_, err = crc.OperatorsV1().OperatorGroups(ns.GetName()).Create(context.TODO(), og, metav1.CreateOptions{})
 		Expect(err).ToNot(HaveOccurred())
 
+		// Wait for the OperatorGroup to be synced and have a status.ServiceAccountRef
+		// before moving on. Otherwise the catalog operator treats it as an invalid OperatorGroup
+		// and fails the InstallPlan
+		Eventually(func() (bool, error) {
+			outOG, err := crc.OperatorsV1().OperatorGroups(ns.GetName()).Get(context.TODO(), og.Name, metav1.GetOptions{})
+			if err != nil {
+				return false, err
+			}
+			fmt.Fprintf(GinkgoWriter, "[DEBUG] Operator Group Status: %+v\n", outOG.Status)
+			return outOG.Status.ServiceAccountRef != nil, nil
+		}).Should(BeTrue())
```

Comment:
```suggestion
		Eventually(func() (*corev1.ObjectReference, error) {
			outOG, err := crc.OperatorsV1().OperatorGroups(ns.GetName()).Get(context.TODO(), og.Name, metav1.GetOptions{})
			if err != nil {
				return nil, err
			}
			return outOG.Status.ServiceAccountRef, nil
		}).ShouldNot(BeNil())
```

It's nice to use the more specific Gomega matcher for this kind of thing instead of converting to a boolean first -- then when the test fails the report will have better information about what went wrong.